includes/controls/repeater.php

```diff
@@ -65,6 +65,7 @@ protected function get_default_settings() {
 			'title_field' => '',
 			'prevent_empty' => true,
 			'is_repeater' => true,
+			'max_items' => 0,
```

Comment:
Since you edited a base file here, can you maybe just do some manual checks in other widgets that are using repeaters to make sure nothing broke?

Reply:
@louiswol94 I tested 
 - Social Icons
 - Icon List
 - Tabs
 - Accordion
 - Toggle
And they are all working as expected, from what I see.